src/api.js

```diff
@@ -23,7 +23,7 @@ async function createRoom() {
     Uncomment the request below to test the "create room" functionality locally.
     Don't forget to comment out the request above, too!
   */
-  // const response = await fetch(`${process.env.REACT_APP_DAILY_API_KEY}/rooms/`, {
```

Comment:
This was giving me errors when I ran it locally then I realized this should be the URL for the daily API, not the API key itself.